### ISSUE TITLE
Running inference methods that are specific to a language

### ISSUE THREAD

**Issue by eerolinna** [MansMeg/bayesbenchr#6]
With our current approach an inference method is essentially any function that satisfies the input/output requirements, so for example `stan_vb` or `stan_sampling`.

A problematic situation is calling a python-only library from R (for example `PyMC`). Lets say a user wants to run the inference task
```json
{
"inference_method": "pymc_nuts",
"posterior": "eight_schools_centered"
}
```

Current situation
- Running this from python works but R gives an error  that is something like "pymc_nuts not found"

Desired situation, either
- R calls out to python to run the inference task
- An error saying that pymc can only be used with python (less preferable)

One solution would be that inference method is not just an function but also some additional data, in this case the data would include `"language": "python"`

**Comment by eerolinna** [comment]
Any comments?

**Comment by MansMeg** [comment]
Note yet. posteriordb needs to be done with the prototype first.

**Comment by eerolinna** [comment]
Can you take a look at these now (so this issue and others in this repo)? It's been almost 2 months.

I think this particular issue is also somewhat relevant for https://github.com/MansMeg/posteriordb/issues/117

**Comment by MansMeg** [comment]
Yes. Ill see exactly how to solve it later. I guess Ill implement som running function in posteriordb first to get it to work. I guess Ill run Python from R.

**Comment by eerolinna** [comment]
I feel like just implementing something is not the best way to go about this. If you can write what you intend to roughly do it will be easier to spot any potential shortcomings before having to spend the effort to actually implement it.
